west.yml

```diff
@@ -215,7 +215,7 @@ manifest:
         - hal
     - name: hal_renesas
       path: modules/hal/renesas
-      revision: 10326518701e25bf336a2eaeb8b5820110e4e6a3
+      revision: pull/34/head
```

Comment:
this should probably point to the merged commit and not on the PR.

Reply:
I changed it to the commit ID